db/connection.js

```diff
@@ -1,10 +1,12 @@
 const { Pool } = require('pg');
 const ENV = process.env.NODE_ENV || 'development';
+console.log(ENV)
```

Comment:
Dont forget to remove un-needed console.log's before pushing your code to github